packages/engine_umbrella/apps/engine/lib/vehicle.ex

```diff
@@ -23,26 +23,36 @@ defmodule Vehicle do
 
   @hour_and_minutes_format ~r/^((?:[01]\d|2[0-3]):[0-5]\d$)/
 
-  validates(:start_address, presence: true)
-  validates([:start_address, :lat], number: [is: true])
-  validates([:start_address, :lon], number: [is: true])
-  validates([:end_address, :lat], number: [is: true])
-  validates([:end_address, :lon], number: [is: true])
-  validates(:earliest_start, format: [with: @hour_and_minutes_format, allow_nil: true])
-  validates(:latest_end, format: [with: @hour_and_minutes_format, allow_nil: true])
+  validates([:start_address, :lat], number: [is: true, if: &(!&1.start_address)])
 
-  validates([:capacity, :weight],
-    by: [function: &is_integer/1, message: "must be an integer"]
-  )
+  # validates([:start_address, :lon],
+  #   number: [is: true, [if: fn map -> !is_nil(map.start_address) end]]
+  # )
 
-  validates([:capacity, :volume], number: [is: true])
+  # validates([:end_address, :lat], number: [is: true, if: &Vehicle.end_address_exists?/1]])
+  # validates([:end_address, :lon], number: [is: true, if: &Vehicle.end_address_exists?/1]])
+
+  # validates([:end_address, :lat], number: [is: true])
+  # validates([:end_address, :lon], number: [is: true])
+  # validates(:earliest_start, format: [with: @hour_and_minutes_format, allow_nil: true])
+  # validates(:latest_end, format: [with: @hour_and_minutes_format, allow_nil: true])
+
+  # validates([:capacity, :weight],
+  #   by: [function: &is_integer/1, message: "must be an integer"]
+  # )
+
+  # validates([:capacity, :volume], number: [is: true])
```

Comment:
@iteadam  Do you know why all validators are commented out? Is it because we want to refactor or we just decided to not have validation currently?